docs/getting-started/install.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Getting Started
+subtitle: An overview of Athena, including how to download and use it, some basic templates and examples, and more.
+---
+
+# Installation
+
+Installing the Athena Framework is as simple as adding the CSS and JS files to your project.
```

Comment:
Let's add some clarification somewhere that Athena-Framework is a standalone framework and not a Bootstrap theme (when installing Athena, users that have Bootstrap installed should replace it with Athena entirely)